instagram-page-react/src/pages/profilePage/reducers/selectors/profilePage.posts.js

```diff
@@ -10,7 +11,15 @@ const getPostListState = compose(
     getProfilePageState
 );
 
-export const getPosts = compose(
+const getPostsList = compose(
     _property('data'),
     getPostListState
 );
+
+export const getPosts = (state, filterFunction = undefined) => {
```

Comment:
filterFunction is an enhancer, we don't need to pass it as a prop
We can create a new selector getReversedPosts by composing the existing getPosts selector with reverseArray

Also, read about createSelector and check if it is applicable here

Reply:
Yes have used `createSelector` here now as it provides the functionality to memoize the output of the `getReversedPosts`.